{"name": "Test login to OrangeHRM with mixed credentials -- @1.3 ", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.SessionNotCreatedException: Message: session not created: This version of ChromeDriver only supports Chrome version 114\nCurrent browser version is 116.0.5845.98 with binary path C:\\Program Files\\Google\\Chrome\\Application\\chrome.exe\nStacktrace:\nBacktrace:\n\tGetHandleVerifier [0x00DDA813+48355]\n\t(No symbol) [0x00D6C4B1]\n\t(No symbol) [0x00C75358]\n\t(No symbol) [0x00C961AC]\n\t(No symbol) [0x00C91EF3]\n\t(No symbol) [0x00C90579]\n\t(No symbol) [0x00CC0C55]\n\t(No symbol) [0x00CC093C]\n\t(No symbol) [0x00CBA536]\n\t(No symbol) [0x00C982DC]\n\t(No symbol) [0x00C993DD]\n\tGetHandleVerifier [0x0103AABD+2539405]\n\tGetHandleVerifier [0x0107A78F+2800735]\n\tGetHandleVerifier [0x0107456C+2775612]\n\tGetHandleVerifier [0x00E651E0+616112]\n\t(No symbol) [0x00D75F8C]\n\t(No symbol) [0x00D72328]\n\t(No symbol) [0x00D7240B]\n\t(No symbol) [0x00D64FF7]\n\tBaseThreadInitThunk [0x76867D59+25]\n\tRtlInitializeExceptionChain [0x772EB79B+107]\n\tRtlClearBits [0x772EB71F+191]\n\n", "trace": "  File \"C:\\Users\\divya\\python_selenium_venv\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\divya\\python_selenium_venv\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\OrangeHRMloginsteps.py\", line 10, in step_impl\n    context.driver = webdriver.Chrome()\n                     ^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\divya\\python_selenium_venv\\Lib\\site-packages\\selenium\\webdriver\\chrome\\webdriver.py\", line 49, in __init__\n    super().__init__(\n  File \"C:\\Users\\divya\\python_selenium_venv\\Lib\\site-packages\\selenium\\webdriver\\chromium\\webdriver.py\", line 54, in __init__\n    super().__init__(\n  File \"C:\\Users\\divya\\python_selenium_venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 206, in __init__\n    self.start_session(capabilities)\n  File \"C:\\Users\\divya\\python_selenium_venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 291, in start_session\n    response = self.execute(Command.NEW_SESSION, caps)[\"value\"]\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\divya\\python_selenium_venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 346, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\divya\\python_selenium_venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 245, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Given I launch Chrome Browser", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.SessionNotCreatedException: Message: session not created: This version of ChromeDriver only supports Chrome version 114\nCurrent browser version is 116.0.5845.98 with binary path C:\\Program Files\\Google\\Chrome\\Application\\chrome.exe\nStacktrace:\nBacktrace:\n\tGetHandleVerifier [0x00DDA813+48355]\n\t(No symbol) [0x00D6C4B1]\n\t(No symbol) [0x00C75358]\n\t(No symbol) [0x00C961AC]\n\t(No symbol) [0x00C91EF3]\n\t(No symbol) [0x00C90579]\n\t(No symbol) [0x00CC0C55]\n\t(No symbol) [0x00CC093C]\n\t(No symbol) [0x00CBA536]\n\t(No symbol) [0x00C982DC]\n\t(No symbol) [0x00C993DD]\n\tGetHandleVerifier [0x0103AABD+2539405]\n\tGetHandleVerifier [0x0107A78F+2800735]\n\tGetHandleVerifier [0x0107456C+2775612]\n\tGetHandleVerifier [0x00E651E0+616112]\n\t(No symbol) [0x00D75F8C]\n\t(No symbol) [0x00D72328]\n\t(No symbol) [0x00D7240B]\n\t(No symbol) [0x00D64FF7]\n\tBaseThreadInitThunk [0x76867D59+25]\n\tRtlInitializeExceptionChain [0x772EB79B+107]\n\tRtlClearBits [0x772EB71F+191]\n\n", "trace": "  File \"C:\\Users\\divya\\python_selenium_venv\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\divya\\python_selenium_venv\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\OrangeHRMloginsteps.py\", line 10, in step_impl\n    context.driver = webdriver.Chrome()\n                     ^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\divya\\python_selenium_venv\\Lib\\site-packages\\selenium\\webdriver\\chrome\\webdriver.py\", line 49, in __init__\n    super().__init__(\n  File \"C:\\Users\\divya\\python_selenium_venv\\Lib\\site-packages\\selenium\\webdriver\\chromium\\webdriver.py\", line 54, in __init__\n    super().__init__(\n  File \"C:\\Users\\divya\\python_selenium_venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 206, in __init__\n    self.start_session(capabilities)\n  File \"C:\\Users\\divya\\python_selenium_venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 291, in start_session\n    response = self.execute(Command.NEW_SESSION, caps)[\"value\"]\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\divya\\python_selenium_venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 346, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\divya\\python_selenium_venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 245, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1692874925000, "stop": 1692874928750}, {"name": "When I open orangeHRM Homepage", "status": "skipped", "start": 1692874928752, "stop": 1692874928752}, {"name": "And Enter username \"admin36\" and password \"admin123\"", "status": "skipped", "start": 1692874928752, "stop": 1692874928752}, {"name": "And Click on login button", "status": "skipped", "start": 1692874928752, "stop": 1692874928752}, {"name": "Then I am on Dashbord page", "status": "skipped", "start": 1692874928752, "stop": 1692874928752}], "parameters": [{"name": "username", "value": "admin36"}, {"name": "password", "value": "admin123"}], "start": 1692874924999, "stop": 1692874928752, "uuid": "c7beafde-5d54-418e-a777-034ee4ef5438", "historyId": "1d14bbda73782a485cbafa7989ec47b3", "fullName": "View Dashboard: Test login to OrangeHRM with mixed credentials", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "View Dashboard"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}